dataquality/models.py

```diff
@@ -1,4 +1,4 @@
-from . import issues
+from crowdsource import issues
```

Comment:
why are issues moved out of this app?

Reply:
Since crowdsource is also using the same model, and crowdsource issues contains dq issues as well, so it makes sense to put crowdsource issues in model of choices.

Reply:
I have made sure I don't change one line in dq, still pass all the tests, as they were failing in the beginning, so I had to see how things are working.

Reply:
I don't understand this reasoning, DQ shouldn't depend on crowdsource

Reply:
Okay, will clear my thoughts about this.